packages/@uppy/react/src/Dashboard.ts

```diff
@@ -51,9 +51,9 @@ class Dashboard<M extends Meta, B extends Body> extends Component<
 
   installPlugin(): void {
     const { uppy, ...options } = {
-      id: 'react:Dashboard',
-      inline: true,
+      id: 'Dashboard',
       ...this.props,
+      inline: true,
```

Comment:
This removes the possibility to override `inline` via props, is it on purpose?

Reply:
It's on purpose and documented in #5206